docs/src/pages/solid.mdx

```diff
@@ -1,58 +1,40 @@
 import { Callout, Steps } from "nextra-theme-docs";
 
-# Getting Started with SolidStart
-
-Make sure you
-[install the packages and get your env variables first](/getting-started)
-
 ## Package Setup
 
 <Steps>
 
 ### Install the packages
 
-import { Tab, Tabs } from "nextra-theme-docs";
-
-{/* prettier-ignore-start */}
-
-<Tabs items={["npm", "yarn", "pnpm"]}>
-  <Tab>
-    ```bash copy 
-    npm install uploadthing @uploadthing/solid solidjs-dropzone
-    ```
-  </Tab>
-  <Tab>
-    ```bash copy 
-    yarn add uploadthing @uploadthing/solid solidjs-dropzone
-    ```
-  </Tab>
-  <Tab>
-    ```bash copy 
-    pnpm install uploadthing @uploadthing/solid solidjs-dropzone
-    ```
-  </Tab>
-</Tabs>
-
-{/* prettier-ignore-end */}
+```bash copy npm2yarn
+npm install uploadthing @uploadthing/solid solidjs-dropzone
+```
 
 ### Add env variables
 
+<Callout>
+  If you don't already have a uploadthing secret key, [sign
+  up](https://uploadthing.com/sign-in) and create one from the
+  [dashboard!](https://uploadthing.com/dashboard)
+</Callout>
+
 ```bash copy
 UPLOADTHING_SECRET=... # A secret key for your app (starts with sk_live_)
 UPLOADTHING_APP_ID=... # Your app id
 ```
 
-<Callout type="warning">
-  If you don't already have a uploadthing secret key, [sign up](https://uploadthing.com/sign-in) and create one from
-  the [dashboard!](https://uploadthing.com/dashboard)
-</Callout>
 </Steps>
 
 ## Set Up A FileRouter
 
 <Steps>
 ### Creating your first FileRoute
 
+<Callout>
+  For more details on how to create a file router, see the [router](/router)
```

Comment:
need to write this page, imo every framework guide don't have to have this. just write 1 page and list the changes to each entrypoint

Reply:
Does `https://docs.uploadthing.com/api-reference/server#file-routes` have most of the information needed here?

Reply:
yea that looks solid!